website/ocular-config.js

```diff
@@ -66,7 +66,24 @@ module.exports = {
     }
   ],
 
-  PROJECTS: [],
+  PROJECTS: [
+    {
+      name: 'deck.gl',
+      url: 'https://deck.gl'
+    },
+    {
+      name: 'luma.gl',
+      url: 'https://luma.gl'
+    },
+    {
+      name: 'react-map-gl',
+      url: 'https://uber.github.io/react-map-gl'
+    },
+    {
+      name: 'react-vis',
```

Comment:
Should we have a link back to the vis.gl main page for all of our frameworks?

Reply:
Sorry was going to update this to add vis.gl before merging, but encountered an issue.  Seems that somewhere deeper in ocular-gatsby the number of items in the drop down is limited to 4.  Will attempt to address in a later PR.

Reply:
Sorry I need to republish. For this issue, I think we can just drop react-vis from the links, not much connection to loaders.gl at the moment.